src/bp/nlu-core/intents/svm-intent-classifier.ts

```diff
@@ -110,9 +110,9 @@ export class SvmIntentClassifier implements IntentClassifier {
         }
       }
 
-      const confidence = 1 / intentNames.length
+      const intent = intentNames[0]
```

Comment:
This change was requested by @EFF in my previous PR

Reply:
I don't recall making this comment but yes good 👍